framework/src/org/checkerframework/common/util/count/AnnotationsCounter.java

```diff
@@ -62,7 +55,7 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
                 @Override
                 public Void visitAnnotation(AnnotationTree node, Void p) {
                     JCAnnotation anno = (JCAnnotation) node;
-                    Name annoName = anno.annotationType.type.tsym.name;
+                    Name annoName = anno.annotationType.type.tsym.getQualifiedName();
```

Comment:
This now uses the fully-qualified name, to allow keeping track of e.g. Nullness annotations in different packages.